src/styles/blocks/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  width: $card-width;
+  padding: 32px $card-padding $card-padding;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: transform $transition-time $transition-linear,
+    color $transition-time $transition-linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rating {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__link {
+    display: block;
+    padding: 11px 0;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-align: center;
+    text-transform: uppercase;
+    color: $white-color;
+
+    background-color: $hover-color;
+    border: 1px solid $hover-color;
+    border-radius: 5px;
+
+    transition: background-color 0.5s, color 0.5s;
+  }
+
+  &:hover &__link {
+    color: $hover-color;
+
+    background-color: $white-color;
+  }
```

You should use :hover only for button, not for card
```suggestion
  &__link {
    display: block;
    padding: 11px 0;

    font-weight: 700;
    font-size: 14px;
    line-height: 16px;
    text-decoration: none;
    text-align: center;
    text-transform: uppercase;
    color: $white-color;

    background-color: $hover-color;
    border: 1px solid $hover-color;
    border-radius: 5px;

    transition: background-color 0.5s, color 0.5s;
  
    &:hover {
      color: $hover-color;

      background-color: $white-color;
    }
  }
```